File: app/src/main/java/ru/p3tr0vich/fuel/views/FuelingTotalViewOnePanel.java
package ru.p3tr0vich.fuel.views;

import android.support.annotation.NonNull;
import android.view.View;
import android.widget.TextView;

import ru.p3tr0vich.fuel.R;

public class FuelingTotalViewOnePanel extends FuelingTotalViewBase {

    private final TextView mAverage;
    private final TextView mCostSum;
    private final TextView mLastConsumption;
    private final TextView mEstimatedMileage;
    private final TextView mEstimatedTotal;

    public FuelingTotalViewOnePanel(@NonNull View view) {
        mAverage = (TextView) view.findViewById(R.id.text_average);
        mCostSum = (TextView) view.findViewById(R.id.text_cost_sum);
        mLastConsumption = (TextView) view.findViewById(R.id.text_last_cons);
        mEstimatedMileage = (TextView) view.findViewById(R.id.text_estimated_mileage);
        mEstimatedTotal = (TextView) view.findViewById(R.id.text_estimated_total);
    }

    @Override
    public void setAverage(float average) {
        mAverage.setText(floatToString(average, false));
    }

    @Override
    public void setCostSum(float costSum) {
        mCostSum.setText(floatToString(costSum, false));
    }

    @Override
    public void setLastConsumption(float lastConsumption) {
        mLastConsumption.setText(floatToString(lastConsumption, false));
    }

    @Override
    public void setEstimatedMileage(float estimatedMileage) {
        mEstimatedMileage.setText(floatToString(estimatedMileage, true));
    }

    @Override
    public void setEstimatedTotal(float estimatedTotal) {
        mEstimatedTotal.setText(floatToString(estimatedTotal, true));
    }
}

File: app/src/main/kotlin/ru/p3tr0vich/fuel/views/FuelingTotalViewOnePanel.kt
package ru.p3tr0vich.fuel.views

import android.view.View
import android.widget.TextView

import ru.p3tr0vich.fuel.R
import ru.p3tr0vich.fuel.utils.UtilsFormat

class FuelingTotalViewOnePanel(view: View) : FuelingTotalViewBase() {

    private val average: TextView = view.findViewById(R.id.text_average)
    private val costSum: TextView = view.findViewById(R.id.text_cost_sum)
    private val lastConsumption: TextView = view.findViewById(R.id.text_last_cons)
    private val estimatedMileage: TextView = view.findViewById(R.id.text_estimated_mileage)
    private val estimatedTotal: TextView = view.findViewById(R.id.text_estimated_total)

    override fun setAverage(value: Float) {
        average.text = UtilsFormat.floatToString(value)
    }

    override fun setCostSum(value: Float) {
        costSum.text = UtilsFormat.floatToString(value)
    }

    override fun setLastConsumption(value: Float) {
        lastConsumption.text = UtilsFormat.floatToString(value)
    }

    override fun setEstimatedMileage(value: Float) {
        estimatedMileage.text = Math.round(value).toString()
    }

    override fun setEstimatedTotal(value: Float) {
        estimatedTotal.text = Math.round(value).toString()
    }
}